src/components/Card.js

```diff
@@ -95,85 +101,98 @@ const MapImg = styled.img`
 `;
 
 const DateTime = styled.div`
-  display: grid;
-  grid-auto-columns: 1fr;
-  grid-template-columns: 0.5fr 1fr 1fr 0.5fr 1fr 1fr 0.5fr;
+  display: flex;
+  justify-content: space-evenly;
 `;
 
 const Date = styled.p`
-  grid-column: 2 / 4;
+  padding: 10px;
+  border: 1px solid #cccccc;
+  border-radius: 10px 10px 10px 10px;
```

Comment:
You can also write `border-radius: 10px` that makes the same! See also line 118, 130, 137, 164, 174 and 196.
And if I see it correctly your styled-components for p tags repeat the following code: 
```
padding: 10px;
  border: 1px solid #cccccc;
  border-radius: 10px 10px 10px 10px;
  text-align: center;
```

I think you could use your Wrapper and set these repeating css for all p tags in this component. That is a suggestion for reducing your code and better readability. :-)